pkgs/development/python-modules/simplesqlite/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, dataproperty
+, mbstrdecoder
+, pathvalidate
+, sqliteschema
+, tabledata
+, typepy
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "SimpleSQLite";
+  version = "1.5.2";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "thombashi";
+    repo = "SimpleSQLite";
+    rev = "v${version}";
+    hash = "sha256-Yr17T0/EwVaOjG+mzdxopivj0fuvQdZdX1bFj8vq0MM=";
+  };
+
+  propagatedBuildInputs = [
+    dataproperty
+    mbstrdecoder
+    pathvalidate
+    sqliteschema
+    tabledata
+    typepy
+  ];
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
```

Comment:
Missing pythonImportsCheck and running tests https://github.com/thombashi/SimpleSQLite/tree/master/test

Reply:
Fixed in https://github.com/NixOS/nixpkgs/pull/299429/commits/7a37a1687b2c503bd94ff4c153da5213d1c8d820